.storybook/preview.tsx

```diff
@@ -14,7 +14,7 @@ const decorators: Decorator[] = [
 		const darkMode = useDarkMode();
 		return (
 			<Provider theme={defaultTheme} colorScheme={darkMode ? 'dark' : 'light'} locale="en-US" height="100vh">
-				<View padding="size-300">
+				<View padding={24} height="calc(100% - 48px)">
```

Comment:
need to calculate the height so that it doesn't include padding. Height of 200px + 24 padding top and bottom makes the total height 248. We want the total height to be 200.